guides/common/modules/con_mirroring-policies-overview.adoc

```diff
@@ -0,0 +1,21 @@
+[id="Mirroring_Policies_Overview_{context}"]
+= Mirroring Policies Overview
+
+Mirroring keeps the local repository exactly in synchronization with the upstream repository. 
+If any content is removed from the upstream repository since the last synchronization, it will be removed from the local repository as well.
```

Comment:
When will it be removed? > "with the next synchronization", I assume. Can you please add this?